apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/soeknadoversikt/kommerBarnetTilgode/EndreVurdering.tsx

```diff
@@ -73,31 +64,32 @@ export const EndreVurdering = ({
         label="Begrunnelse"
         hideLabel={false}
         placeholder="Forklar begrunnelsen"
-        value={kommentar}
+        value={begrunnelse}
         onChange={(e) => {
-          setKommentar(e.target.value)
-          kommentar.length > 10 && setBegrunnelseError(undefined)
+          setBegrunnelse(e.target.value)
+          begrunnelse.length > 10 && setBegrunnelseError(undefined)
         }}
         minRows={3}
         size="small"
         error={begrunnelseError ? begrunnelseError : false}
       />
       <Button
+        loading={isPending(kommerBarnetTilGode)}
         style={{ marginTop: '10px' }}
         variant={'primary'}
         size={'small'}
         onClick={() => lagreBegrunnelseKlikket()}
       >
         Lagre
       </Button>
-      <Button
-        style={{ marginTop: '10px', marginLeft: '10px' }}
-        variant={'secondary'}
-        size={'small'}
-        onClick={() => reset()}
-      >
+      <Button style={{ marginLeft: '10px' }} variant={'secondary'} size={'small'} onClick={setRedigeringsModusFalse}>
         Avbryt
       </Button>
+      {isFailure(kommerBarnetTilGode) && <ApiErrorAlert variant="error">Kunne ikke lagre vurdering</ApiErrorAlert>}
```

Comment:
Skal vi skille mellom feil på teknisk lagringsfeil og at behandlingen har feil status?

Reply:
Det tenker jeg er fornuftig i alle fall.  Burde kanskje brukt en egen http status code for det også, egentlig?

Reply:
Jepp det synes jeg.

Reply:
@aryani1 Hva tenker du?

Reply:
Er lurt det. Må bli enige om hvordan vi håndterer det bare, kan lage en postkasse-lapp på det med feilmeldinger til bruker.